### ISSUE TITLE
Discolored pixels

### ISSUE THREAD

**Issue by Aiz0** [DanganMachin/Fem-Jokers-v.1.1.0#1]
There seem to be a lot of sprites with discolored pixels, did you use a compressed version of the images?

Here are some examples where i selected by color with 0 tolerance.

chaos the clown: the emoji in place of the "o" in "joker" have a lot of discoloration
![2024-07-30_T00:24:14](https://github.com/user-attachments/assets/9807c551-1f1e-4dde-b47b-3fc3647df7c5)

odd todd
![image](https://github.com/user-attachments/assets/d015c8be-93bf-4282-9f27-a228bb465213)
even steven
![image](https://github.com/user-attachments/assets/b083b1cf-95d6-4071-b396-d9b9a5164c13)

vampire: dark pixels near other colors
![image](https://github.com/user-attachments/assets/2aa9f349-0c58-430d-93d3-5622d3b5e628)

Lucky cat, vagabond, Baron, and a few others don't seem to have this problem.

**Comment by DanganMachin** [comment]
I don't know, I asked someone on discord to help me but those in the sprite sheet, are you sure these 4 are the only to have that problem ? Because I am preparing an update which replace those sprite with corrected ones (the joker on the bottom right is reversed).

**Comment by Aiz0** [comment]
I did a quick test and the jokers that seem to have this problem are:
1. Chaos
2. Mime
3. Even steven
4. Odd todd
5. Sixth sense
6. Vampire

**Comment by Aiz0** [comment]
Only other thing I've found is that baseball card seems to have some out of place pixels in the top right and bottom left corners

**Comment by DanganMachin** [comment]
Yep, it makes sense that these are the ones causing problem, it should be fixed in the next update, if not tell me ! (:
